src/Chapter4/2_test_using_for_loop.cpp

```diff
@@ -9,7 +9,7 @@ class EmployeeTestFixture : public ::testing::Test {
 
 // Test net bonus works fine for different number of years.
 TEST_F(EmployeeTestFixture, NetBonusIsCorrectForDifferentYears) {
-    auto input = std::vector<int>{5, 15};
+    auto input = std::vector<unsigned int>{5, 15};
     auto expected_output = std::vector<int>{2000, 3000};
```

Comment:
Shouldn't outputs be `float`?

Reply:
Yes! Good spot.

And we should be using `EXPECT_DOUBLE_EQ` below instead.